backend/src/routes/__test__/person.route.test.ts

```diff
@@ -1,69 +1,205 @@
 import httpStatus from 'http-status';
 import databaseOperations from '../../utils/test/db-handler';
+
 import { PersonModel } from '../../models/person.model';
 import app from '../../server';
 import testUtils from '../../utils/test/test-utils';
+import "dotenv/config";
+import personService from '../../services/person.service';
+import userService from '../../services/user.service';
 
 const supertest = require('supertest');
 
 let token;
 
 beforeAll(async () => {
   token = await testUtils.generateTestAuthToken();
-
   databaseOperations.connectDatabase();
 });
 afterEach(async () => databaseOperations.clearDatabase());
 afterAll(async () => databaseOperations.closeDatabase());
 
-const requestPersonData:PersonModel = {
-  first_name: 'testFirstName',
-  last_name: 'testLastName',
-  interests: ['a', 'b'],
-  organisation: 'testorg',
+const reqUserData = {
+  first_name: 'Bing',
+  last_name: 'Bong',
+  encounters: [] as any,
+  persons: [] as any,
+}
+
+const person1Data: PersonModel = {
+  first_name: 'Ping',
+  last_name: 'Pong',
+  interests: ['video games', 'hockey'],
+  organisation: 'helloc',
   time_updated: new Date('2022-01-01'),
-  how_we_met: 'testmet',
+  how_we_met: 'Hockey club',
   birthday: new Date('2002-12-12'),
   encounters: [] as any,
   first_met: new Date('2022-01-01'),
-  gender: "other",
+  gender: "male",
+  location: "Auckland",
   image: null as any,
-  location: 'Someplace',
   social_media: null as any
 };
 
-describe('person ', () => {
-  it('can be created correctly', async () => {
-    await supertest(app).post('/api/persons')
+const person2Data: PersonModel = {
+  first_name: 'Adam',
+  last_name: 'Bong',
+  interests: ['badminton', 'golf'],
+  organisation: 'helloc',
+  time_updated: new Date('2022-02-23'),
+  how_we_met: 'Skype',
+  birthday: new Date('2001-07-16'),
+  encounters: [] as any,
+  first_met: null as any,
+  gender: "male",
+  image: null as any,
+  location: null as any,
+  social_media: null as any
+}
+
+const person3Data: PersonModel = {
+  first_name: 'Billy',
+  last_name: 'John',
+  interests: ['surfing', 'cooking'],
+  organisation: 'an organisation',
+  time_updated: new Date('2022-02-23'),
+  how_we_met: 'At the park',
+  birthday: new Date('2001-07-16'),
+  encounters: [] as any,
+  first_met: null as any,
+  gender: "male",
+  image: null as any,
+  location: null as any,
+  social_media: null as any
+}
+
+describe('Creating a person', () => {
```

Comment:
Could we change this `describe` name to include the endpoint it's testing (similar to how some of the other tests are written)? Just for the sake of clarity.

Reply:
No worries, should be fixed now